common/JackEngine.h

```diff
@@ -163,6 +163,9 @@ class SERVER_EXPORT JackEngine : public JackLockAble
         int GetClientNameForUUID(const char *uuid, char *name_res);
         int ReserveClientName(const char *name, const char *uuid);
         int ClientHasSessionCallback(const char *name);
+
+		// fMetadata needs to be public for JackNetManager.cpp to refernce to it
+		JackMetadata fMetadata;
```

Comment:
Would it be possible to friend-class JackNetManager (don't know the exact class name) instead?

Reply:
that would be a great idea, this f-prefixed instances have been annoying me quite a lot, this here https://github.com/hlolli/jack2/blob/windows/msvc-with-waf-fix/common/wscript#L360-L363 I couldn't after million attemts get to compile. The symbol is visible and all looks right, but the linker wont find the symbol of fInstance.